README.md

```diff
@@ -114,8 +186,11 @@ val flightNumber by columnOf(10045.0, Double.NaN, 10065.0, Double.NaN, 10085.0)
 val recentDelays by columnOf("23,47", null, "24, 43, 87", "13", "67, 32")
 val airline by columnOf("KLM(!)", "{Air France} (12)", "(British Airways. )", "12. Air France", "'Swiss Air'")
 
-// create dataframe
+// create the dataframe
```

Comment:
I'd say "a" here. But it's a comment, which are better when they're compact, so maybe it could be left as is.